src/components/App.js

```diff
@@ -9,12 +9,20 @@ class App extends Component {
   constructor() {
     super()
     this.state = {
-      movies: movieData.movies, 
+      movies: [], 
       pageDetails: false,
-      currentMovie: ''
+      currentMovie: '',
+      error: '',
     }
   }
 
+  componentDidMount = () => {
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+    .then(response => response.json())
+    .then(data => this.setState({movies: data.movies}))
+    .catch(err => this.setState({error: "Something we wrong, Please try again later."}))
+  }
+
   pageDetailsUpdate = (id) => {
     let status = false;
     if (this.state.pageDetails === false) {
```

Comment:
This seems like you are toggling `this.state.pageDetails`. If so, then consider utilizing the previous state in `setState`: https://reactjs.org/docs/state-and-lifecycle.html#state-updates-may-be-asynchronous

Then you might be able to use `!` to toggle `true` and `false`.

Reply:
Overall, that would simplify the logic here.